File: src/main/java/org/example/eshop/controller/AdminViewController.java
package org.example.eshop.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
@RequestMapping("/admin")
public class AdminViewController {

    @GetMapping("/login")
    public String loginPage() {
        return "admin/login";
    }

    @GetMapping("/dashboard")
    public String dashboardPage() {
        return "admin/dashboard";
    }

    @GetMapping("/orders")
    public String ordersPage() {
        return "admin/orders";
    }

    @GetMapping("/products")
    public String productsPage() {
        return "admin/products";
    }
}

File: src/main/kotlin/org/example/eshop/controller/AdminViewController.kt
package org.example.eshop.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping

@Controller
@RequestMapping("/admin")
class AdminViewController {

    @GetMapping("/login")
    fun loginPage(): String {
        return "admin/login"
    }

    @GetMapping("/dashboard")
    fun dashboardPage(): String {
        return "admin/dashboard"
    }

    @GetMapping("/orders")
    fun ordersPage(): String {
        return "admin/orders"
    }
}